opengraph/dom-test/README.md

```diff
@@ -0,0 +1,9 @@
+# OpenGraph Standards test
```

Comment:
I'd add a few more sections to the README 
## Requirements
all requirements will be installed by the run file (see Test Execution section below)
This test runs against remote URLs and doesn't require a browser
## Test Execution

~~~
$ cd services-test/opengraph/dom-test
$ run
~~~